packages/sui-ssr/server/dynamicRendering/index.js

```diff
@@ -36,6 +37,17 @@ export default function dynamicRendering(fallback, dynamicsURLS = []) {
             'rel="stylesheet" media="only x" as="style" onload="this.media=\'all\'"'
           )
         )
+        .pipe(
+          replace(
+            HEAD_CLOSING_TAG,
+            `
+        <script>
+        /*! loadCSS. [c]2017 Filament Group, Inc. MIT License */
```

Comment:
what about saving this in a const called PREFETCH_POLYFILL and use it? should make this whole stuff more readable...